msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerMSALController.java

```diff
@@ -161,18 +165,16 @@ private static BrokerRequest getBrokerRequestForSilent(MSALAcquireTokenSilentOpe
         BrokerRequest request = new BrokerRequest();
         request.setApplicationName(parameters.getAppContext().getPackageName());
         request.setAuthority(parameters.getAuthority().getAuthorityURL().toString());
-        //request.setClaims("");
         request.setClientId(parameters.getClientId());
         request.setCorrelationId(DiagnosticContext.getRequestContext().get(DiagnosticContext.CORRELATION_ID));
-        //request.setExtraQueryStringParameter();
         request.setForceRefresh(parameters.getForceRefresh());
         request.setLoginHint(parameters.getAccount().getUsername());
         request.setName(parameters.getAccount().getUsername());
         request.setUserId(parameters.getAccount().getHomeAccountId());
-        //request.setPrompt(parameters.get);
         //TODO: This should be the broker redirect URI and not the non-broker redirect URI
         request.setRedirect(parameters.getRedirectUri());
         request.setScope(StringUtil.join(' ', parameters.getScopes()));
+        request.setClaims(ClaimsRequest.getJsonStringFromClaimsRequest(parameters.getClaimsRequest()));
```

Comment:
`ClaimsRequests` could delegate de/serialization to `ClaimsRequestSerializer` and `ClaimsRequestDeserializer`.

I can also buy that this should stay as-is since this is by nature _slightly_ different than the `QueryParamsAdapter` since, unlike the `List` we use to hold params, we have an actual `class` which can handle its own serialization.

Reply:
`ClaimsRequestSerializer` and `ClaimsRequestDeserializer` are package protected and cannot be accessed in ad-accounts. So leaving it as it to get it from `ClaimsRequests`